engine/src/main/java/org/terasology/rendering/dag/WireframeStateChange.java

```diff
@@ -15,11 +15,12 @@
  */
 package org.terasology.rendering.dag;
 
+
 /**
  *
  */
-public class WireframeStateChange extends AbstractStateChange<Boolean> {
-    WireframeStateChange(Boolean value) {
+public class WireframeStateChange extends AbstractStateChange<Integer> {
```

Comment:
Sorry for being obsessed about it. Let me try to put an FBO binding operation in my next commits.